stl/inc/__msvc_int128.hpp

```diff
@@ -1407,8 +1407,9 @@ class numeric_limits<_Signed128> : public _Num_int_base {
         return 0;
     }
 
-    static constexpr int digits   = 127;
-    static constexpr int digits10 = 38;
+    static constexpr bool is_signed = true;
+    static constexpr int digits     = 127;
+    static constexpr int digits10   = 38;

```

Comment:
Let's follow the order in which these members are declared in the Standard:
```suggestion
    static constexpr int digits     = 127;
    static constexpr int digits10   = 38;
    static constexpr bool is_signed = true;
```